<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_cameraConfigGroupBox">
     <property name="title">
      <string>Camera config</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="m_widthLabel">
          <property name="text">
           <string>Resolution width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_heightLabel">
          <property name="text">
           <string>Resolution height:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="m_brightnessLabel">
          <property name="text">
           <string>Brightness:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="m_sharpnessLabel">
          <property name="text">
           <string>Sharpness:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="m_contrastLabel">
          <property name="text">
           <string>Contrast:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="m_isoLabel">
          <property name="text">
           <string>ISO:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Saturation:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="m_heightSpinBox">
            <property name="minimum">
             <number>240</number>
            </property>
            <property name="maximum">
             <number>4800</number>
            </property>
            <property name="singleStep">
             <number>240</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSlider" name="m_sharpnessSlider">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_sharpnessSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSlider" name="m_brightnessSlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_brightnessSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSlider" name="m_contrastSlider">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_contrastSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSlider" name="m_isoSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>8</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_isoSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>800</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QSlider" name="m_saturationSlider">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_saturationSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_widthSpinBox">
          <property name="minimum">
           <number>320</number>
          </property>
          <property name="maximum">
           <number>6400</number>
          </property>
          <property name="singleStep">
           <number>320</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_applyPushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_revertPushButton">
       <property name="text">
        <string>Revert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_brightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_brightnessSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_brightnessSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_brightnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sharpnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_sharpnessSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sharpnessSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_sharpnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_contrastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_contrastSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_contrastSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_contrastSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_isoSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnISOSliderValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_isoSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnISOSpinBoxValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_saturationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_saturationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_saturationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_widthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnWidthSpinBoxValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_heightSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnHeightSpinBoxValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnOkButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_applyPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnApplyButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_revertPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnRevertButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnISOSliderValueChanged(int)</slot>
  <slot>OnISOSpinBoxValueChanged(int)</slot>
  <slot>OnWidthSpinBoxValueChanged(int)</slot>
  <slot>OnHeightSpinBoxValueChanged(int)</slot>
  <slot>OnApplyButtonClicked()</slot>
  <slot>OnRevertButtonClicked()</slot>
  <slot>OnOkButtonClicked()</slot>
 </slots>
</ui>
